source/served/extension.d

```diff
@@ -991,9 +1042,43 @@ void onDidSaveDocument(DidSaveTextDocumentParams params)
 	}
 }
 
+@protocolNotification("workspace/didChangeWatchedFiles")
+void onDidChangeWatchedFiles(DidChangeWatchedFilesParams params)
+{
+	foreach (c; params.changes)
+	{
+		trace("watched file did change: ", c);
```

Comment:
you should check if `c` is actually the filename you look for and not some other file like profilegc.log